src/coreclr/src/System.Private.CoreLib/src/System/Reflection/Emit/MethodBuilder.cs

```diff
@@ -135,14 +135,14 @@ public sealed class MethodBuilder : MethodInfo
 
         #region Internal Members
 
-        internal void CheckContext(params Type[]?[]? typess)
+        internal static void CheckContext(params Type[]?[]? typess)
```

Comment:
Nit: can we fix the argument name as long as we're tweaking this?

Reply:
Cleaned up